config/locales/rdr.en.yml

```diff
@@ -18,6 +18,7 @@ en:
     unexpected_multiple_results: "Unexpected multiple results for %{identifier}"
     user_not_found: "User %{user_key} could not be found"
     not_found: "%{target} could not be found"
+    home_link: "Back to Research Data Repository Home"
```

Comment:
@ad96 If you haven't already, you may want to check with product owners about this wording; e.g., whether to include the term "Repository" or not.  I think maybe we're generally not including it in user-facing text but maybe I'm wrong.

Reply:
@ad96 Never mind ... I just saw the new home page uses Research Data Repository so I think your translation is good as is.

Reply:
Thanks, good idea! I've added a comment on the JIRA to get their input.